WorkoutTest/DayViewController.swift

```diff
@@ -29,6 +34,10 @@ class DayViewController: UIViewController {
     
   }
   
+  @objc func openActionSheet() {
+    showActionSheet()
+  }
```

Comment:
what is the point of having a function that just calls another function?) 
you could just have a single one - `func showActionSheet()` and use it as a selector in `trashButton`